sdks/java/io/cdap/src/test/java/org/apache/beam/sdk/io/cdap/CdapIOTest.java

```diff
@@ -154,11 +171,45 @@ public void testReadingData() {
     for (int i = 1; i < EmployeeInputFormat.NUM_OF_TEST_EMPLOYEE_RECORDS; i++) {
       expected.add(KV.of(String.valueOf(i), EmployeeInputFormat.EMPLOYEE_NAME_PREFIX + i));
     }
-    PCollection<KV<String, String>> actual = p.apply("ReadTest", read);
+    PCollection<KV<String, String>> actual = p.apply("ReadBatchTest", read);
     PAssert.that(actual).containsInAnyOrder(expected);
     p.run();
   }
 
+  @Test
+  public void testReadingDataStreaming() {
```

Comment:
To clarify, this is just using a streaming plugin (not running a streaming ppipeline) ? We should name the test appropriately.

Reply:
Renamed